backend/de.metas.business/src/main/sql/postgresql/system/20-de.metas.business/5671210_sys_gh14346_customize_material_receipt_window.sql

```diff
@@ -0,0 +1,11 @@
+-- Field: Wareneingang_OLD -> Wareneingang -> Referenz
+-- Column: M_InOut.POReference
+-- 2023-01-11T10:57:13.070Z
+UPDATE AD_Field SET AD_Reference_ID=10,Updated=TO_TIMESTAMP('2023-01-11 11:57:13.069','YYYY-MM-DD HH24:MI:SS.US'),UpdatedBy=100 WHERE AD_Field_ID=3502
+;
+
+-- Column: M_InOut.POReference
+-- 2023-01-11T10:58:14.074Z
+UPDATE AD_Column SET IsAlwaysUpdateable='Y',Updated=TO_TIMESTAMP('2023-01-11 11:58:14.074','YYYY-MM-DD HH24:MI:SS.US'),UpdatedBy=100 WHERE AD_Column_ID=3799
```

Comment:
just to make sure: do we have a requirement to make POReference editable even if the material receipt is completed?

I think yes, but pls just confirm.

Reply:
I have it for the customer, but i thought it will needed in metasfresh too